lib/tasks/message_queues.rake

```diff
@@ -57,4 +57,44 @@ namespace :message_queues do
       exit 1
     end
   end
+
+  desc "Run workers to consume subscriber list updates for major change messages from rabbitmq"
+  task :update_subscriber_list_details_major_consumer do
+    logger.info "Bound to exchange #{exchange_name} on major change"
+    begin
+      GovukError.configure
+      EmailAlertService.services(:redis)
+      GovukMessageQueueConsumer::Consumer.new(
+        queue_name: "update_subscriber_list_details_major",
+        processor: SubscriberListDetailsUpdateProcessor.new(logger),
+        logger: logger,
+      ).run
+    rescue SignalException => e
+      logger.info "Signal Exception: #{e}"
+      exit 1
+    rescue StandardError => e
+      logger.info "Error: #{e}"
+      exit 1
+    end
+  end
+
+  desc "Run workers to consume subscriber list updates for minor change messages from rabbitmq"
+  task :update_subscriber_list_details_minor_consumer do
+    logger.info "Bound to exchange #{exchange_name} on minor change"
+    begin
+      GovukError.configure
+      EmailAlertService.services(:redis)
+      GovukMessageQueueConsumer::Consumer.new(
+        queue_name: "update_subscriber_list_details_minor",
+        processor: SubscriberListDetailsUpdateProcessor.new(logger),
+        logger: logger,
+      ).run
+    rescue SignalException => e
+      logger.info "Signal Exception: #{e}"
+      exit 1
+    rescue StandardError => e
+      logger.info "Error: #{e}"
+      exit 1
+    end
+  end
```

Comment:
Since we now have 4 almost-identical chunks of code, I think this file would benefit from a method which abstracts over this.